vtex/profile-form
# Add disabled prop into input rules

#### What is the purpose of this pull request?
As the title says.

#### What problem is this solving?
It was not able to disable an input of the profile form.

#### How can it be tested?
Go to `account/profile` and click in the edit info button. So, for test, you will see that every input is disabled.
[Click here to access the workspace.](https://waza2--recorrenciaqa.myvtex.com/account#/profile)

#### Types of changes
